src/megaapi_impl.cpp

```diff
@@ -15033,9 +15033,10 @@ void MegaApiImpl::getregisteredcontacts_result(error e, vector<tuple<string, str
                 for (const auto& row : *data)
                 {
                     vector<char*> list;
-                    list.emplace_back(MegaApi::strdup(std::get<0>(row).c_str()));
-                    list.emplace_back(MegaApi::strdup(std::get<1>(row).c_str()));
-                    list.emplace_back(MegaApi::strdup(std::get<2>(row).c_str()));
+                    forEach(row, [&list](const string& value)
```

Comment:
Before committing this kind of changes, it would be nice if they were discussed with the team. Personally, for a 3 elements list, I find the old syntax much more readable than the new one.